conf/web.xml

```diff
@@ -3948,6 +3948,10 @@
         <extension>trm</extension>
         <mime-type>application/x-msterminal</mime-type>
     </mime-mapping>
+    <mime-mapping>
+        <extension>ts</extension>
+        <mime-type>video/mp2t</mime-type>
```

Comment:
The canonical name seems to be `[video/MP2T](https://www.iana.org/assignments/media-types/video/MP2T)` (I know that it is case-insensitive)